File: Patrones/Inmutable/Java/Inmutable/src/example/Color.java
package example;

public class Color {
	private int R;
	private int G;
	private int B;
	
	public Color(int r, int g, int b){
		this.R = r;
		this.G = g;
		this.B = b;		
	}
	
	public int getR(){
		return this.R;
	}
	
	public int getG(){
		return this.G;
	}
	
	public int getB(){
		return this.B;
	}
	
	public String toString(){
		return this.R + "," + this.G + "," + this.B;
	}
}

File: Patrones/Inmutable/C++/src/Color.cpp
#include <Color.h>

Color::Color(int r, int g, int b){
	this->R = r;
	this->G = g;
	this->B = b;
}

int Color::getR(){
	return this->R;
}

int Color::getG(){
	return this->G;
}

int Color::getB(){
	return this->B;
}

string Color::toString(){
	return to_string(this->R) + "," + to_string(this->G) + "," + to_string(this->B);
}